code/game/objects/items/documents.dm

```diff
@@ -115,3 +221,218 @@
+		. += "Two weird settings dials are visible within the battery compartment."
+
+/obj/item/inspector/clown/examine_more(mob/user)
+	if(cell_cover_open)
+		var/list/msg =  list("<span class='notice'>Both setting dials are flush with the surface of the battery compartment, and seem to be impossible to move with bare hands.</span>")
+		msg += "\t<span class='info'>The first dial is labeled \"SPEED\" and looks a bit like a <strong>screw</strong> head.</span>"
+		msg += "\t<span class='info'>The second dial is labeled \"SOUND\". It has four small holes in it. Perhaps it can be turned with a fork?</span>"
+		return msg
+	return ..()
+
+/obj/item/inspector/clown/proc/cycle_print_time(mob/user)
+	var/message = "You turn the screw-like dial, setting the device's scanning speed to "
+	if(print_time == 1 SECONDS)
+		print_time = 5 SECONDS
+		message += "SLOW."
+	else
+		print_time = 1 SECONDS
+		message += "LIGHTNING FAST."
+	balloon_alert(user, message)
+
+/obj/item/inspector/clown/proc/cycle_sound(mob/user)
+	print_sound_mode++
+	if(print_sound_mode > max_mode)
+		print_sound_mode = INSPECTOR_PRINT_SOUND_MODE_NORMAL
+	balloon_alert(user, "You turn the dial with holes in it, setting the device's bleep setting to [mode_names[print_sound_mode]] mode.")
+
+/obj/item/inspector/clown/create_slip()
+	var/obj/item/paper/fake_report/slip = new(get_turf(src))
+	slip.generate_report(get_area(src))
+
+/**
+ * # Bananium HONK-spect scanner
+ *
+ * An upgraded version of the fake N-spect scanner
+ *
+ * Can print things way faster, at full power the reports printed by this will destroy
+ * themselves and leave water behind when folding is attempted by someone who isn't an
+ * origami master. Printing at full power costs INSPECTOR_POWER_USAGE_HONK cell units
+ * instead of INSPECTOR_POWER_USAGE_NORMAL cell units.
+ */
+/obj/item/inspector/clown/bananium
+	name = "\improper Bananium HONK-spect scanner"
+	desc = "Honkmother-blessed inspection device. Performs inspections according to Clown protocols when activated, then \
+			prints a clowncrypted report regarding the maintenance of the station. Hard to replace."
+	icon_state = "bananium_inspector"
+	w_class = WEIGHT_CLASS_SMALL
+	max_mode = BANANIUM_CLOWN_INSPECTOR_PRINT_SOUND_MODE_LAST
+	///How many more times can we print?
+	var/paper_charges = 200
+	///Max value of paper_charges
+	var/max_paper_charges = 500
+	///How much charges are restored per paper consumed
+	var/charges_per_paper = 25
+
+/obj/item/inspector/clown/bananium/proc/check_settings_legality()
+	if((print_sound_mode == INSPECTOR_PRINT_SOUND_MODE_NORMAL)&&(print_time < 1 SECONDS))
+		if(cell.use(power_to_speak))
+			say("Setting combination forbidden by Geneva convention revision CCXXIII selected, reverting to defaults")
+		print_time = 5 SECONDS
+		print_sound_mode = INSPECTOR_PRINT_SOUND_MODE_CLASSIC
+		power_per_print = INSPECTOR_POWER_USAGE_NORMAL
+
+/obj/item/inspector/clown/bananium/attackby(obj/item/I, mob/user, params)
+	..()
```

Comment:
```suggestion
	. = ..()
```
By default and unless you have an explicit reason not to, preserve parent call return, even if `null`.